test/cctest/test_path.cc

```diff
@@ -45,3 +49,49 @@ TEST_F(PathTest, PathResolve) {
             "/foo/tmp.3/cycles/root.js");
 #endif
 }
+
+TEST_F(PathTest, ToNamespacedPath) {
+  const v8::HandleScope handle_scope(isolate_);
+  Argv argv;
+  Env env{handle_scope, argv, node::EnvironmentFlags::kNoBrowserGlobals};
+#ifdef _WIN32
+  BufferValue data(isolate_,
+                   v8::String::NewFromUtf8(isolate_, "").ToLocalChecked());
+  ToNamespacedPath(*env, &data);
+  EXPECT_EQ(data.ToStringView(), "");  // Empty string should not be mutated
+  // If the resolved name is no more than 2 characters, then path
```

Comment:
@lemire Should we re-enable these tests?

Reply:
Do you understand my comment? 

Consider that

```
path.toNamespacedPath("C:")
```

does not generate a 2-byte path. You need a 2-byte path *after* the resolution.

Reply:
So I consider that the test was wrong, and I have disabled it with a long comment.

I think that if you reenable it, it will just fail again... but because you are not testing what you thought you were testing.

Reply:
Obviously, that's my understanding and I could always be wrong, but do review my comment.